### ISSUE TITLE
Texture glitch when playing model animation.

### ISSUE THREAD

**Issue by FlexYourBrain** [defold/defold-pbr#2]
Seeing texture glitches when playing animation. The inside of the cup and on the plate when the plate lands and settles.

Captured:
![Texture_tear](https://github.com/defold/defold-pbr/assets/28400489/c15da1cc-7607-4db6-8b5d-995420271ddc)

Tested with defold 1.4.8 on win7 / win10 / win11

Graphics cards:
GTX-960 / GTX 1060 / RTX 4070 all same results. 

Here is a repro.
[CupO_Defold-pbr.zip](https://github.com/defold/defold-pbr/files/12381033/CupO_Defold-pbr.zip)

**Comment by einarnordengren** [comment]
I think this was fixed in Defold 1.6.3 (or according to the release notes 1.6.4). Looks like the tangents aren't calculated correctly which was fixed for skinned meshes here: [https://github.com/defold/defold/pull/8338](https://github.com/defold/defold/pull/8338)

**Comment by FlexYourBrain** [comment]
Thanks I will have to check it out and close the issue once confirmed.

**Comment by FlexYourBrain** [comment]
> Looks like the tangents aren't calculated correctly which was fixed for skinned meshes here: [defold/defold#8338](https://github.com/defold/defold/pull/8338)

You are correct the angles now render correctly. This issue is solved. Thanks!
